src/fastspell/fastspell.py

```diff
@@ -127,7 +158,7 @@ def load_hunspell_dicts(self):
         self.similar = []
         for sim_entry in self.similar_langs:
             if sim_entry.split('_')[0] == self.lang:
-                self.similar.append(self.similar_langs[sim_entry])
+                self.similar.append(self.similar_langs[sim_entry] + [self.lang])
```

Comment:
This is also unrelated to the AI model of course (this part is not used for it).
I introduced this change to make sure the target language was also considered for spellchecking. Basically, to avoid having to remember to do things like https://github.com/mbanon/fastspell/blob/70d832a2840d9a9f66d873beeaabf643889faa4f/src/fastspell/config/similar.yaml#L4 (where you have to remember to repeat the code of the target language in the similar list: "az: [tr, az]"). I think it is a bit of a footgun for users of the library: as I was testing something like "it: [es, pt, fr, en]", I didn't realize at first without looking at the code that I had to repeate "it".

Interesting thing I didn't expect: in the lingua-py dataset with Italian, when fastText makes a mistake it most often detects Italian as English (https://github.com/pemistahl/lingua-py/blob/534528d99fd0914bdeda92138a58f6073e1067b9/accuracy-reports/fasttext/Italian.txt#L1).

Reply:
Let me know if you want to land this change and I'll open a PR for it.

Reply:
I opened #20.